docs/user-guide/metadata.md

```diff
@@ -281,3 +281,31 @@ jobs:
           meta set notification.slack.component.channels "fail_channel, prod_channel"
 ```
 In the above example a Slack notification failure message will be send to channels `fail_channel` and `prod_channel` instead of `main_channel`. All other jobs in this pipeline would still post to `main_channel`.
+
+#### Job-based `minimized`setting
```

Comment:
can you add an entry for this in the Table of Contents at the top of this file?

Reply:
Done.